Mathlib/Tactic/Positivity/Core.lean

```diff
@@ -202,16 +202,16 @@ def normNumPositivity (e : Q($α)) : MetaM (Strictness zα pα e) := catchNone d
   | .isRat _i q n d p =>
     let _a ← synthInstanceQ q(LinearOrderedRing $α)
     assumeInstancesCommute
-    have p : Q(NormNum.IsRat $e $n $d) := p
+    have p : Q(@NormNum.IsRat _ StrictOrderedRing.toRing $e $n $d) := p
     if 0 < q then
       haveI' w : decide (0 < $n) =Q true := ⟨⟩
-      pure (.positive q(pos_of_isRat $p $w))
+      pure (.positive (q(pos_of_isRat $p $w) : Expr))
```

Comment:
Using `Expr` in this way is an antipattern; usually you should be adding an extra `$x =Q $y` variable to the context to document what defeq you are assuming (which `assumeInstancesCommute` often does for you